<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesPage</class>
 <widget class="QWidget" name="FilesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QToolButton" name="viewBtn">
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveBtn">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="resources/tracks.qrc">
         <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="filesTable">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>19</number>
      </attribute>
      <column>
       <property name="text">
        <string>Файл</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Изменен</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Путь</string>
       </property>
      </column>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabDescr">
       <attribute name="title">
        <string>Описание</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="fileDescrEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPreview">
       <attribute name="title">
        <string>Просмотр</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="previewFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSaveFileAs">
   <property name="text">
    <string>Сохранить файл...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/tracks.qrc"/>
  <include location="resources/tracks.qrc"/>
 </resources>
 <connections/>
</ui>
